src/common/meta/organization.rs

```diff
@@ -23,8 +23,26 @@ pub const THRESHOLD: i64 = 9383939382;
 
 #[derive(Serialize, Deserialize, ToSchema, Clone, Debug)]
 pub struct Organization {
+    #[serde(default)]
     pub identifier: String,
-    pub label: String,
+    pub name: String,
+    #[serde(default)]
+    pub org_type: String,
+}
+
+#[cfg(feature = "enterprise")]
+#[derive(Serialize, Deserialize, ToSchema, Clone, Debug)]
+pub struct OrganizationInvites {
+    #[serde(default)]
+    pub invites: Vec<String>, // user emails
+    pub role: UserRole,
+}
```

Comment:
looks like this struct needs to be added to OpenAPI schema here
`src/handler/http/router/openapi.rs`

Reply:
Nice catch! will fix the invites specific changes in a new pr. For now, I will not expose organization invitations api in this pr.